health-services/project-factory/src/server/api/genericApis.ts

```
@@ -1456,11 +1544,31 @@
+  const MdmsCriteria : MDMSModels.MDMSv2RequestCriteria= {
+    MdmsCriteria: {
+      tenantId: tenantId,
+      schemaCode: "HCM-ADMIN-CONSOLE.schemas",
+      uniqueIdentifiers: [
+        `${type}`
+      ]
+    }
+  };
+  const response = await searchMDMSDataViaV2Api(MdmsCriteria, true);
+  if (!response?.mdms?.[0]?.data) {
+    throwError("COMMON", 500, "INTERNAL_SERVER_ERROR", "Error occured during schema search for " + type);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)